examples/golang-todo-rest-crud/.github/workflows/ci.yml

```diff
@@ -0,0 +1,41 @@
+name: ci
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+jobs:
+  golangci-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.19
+      - uses: actions/checkout@v3
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v3
+        with:
+          version: latest
+  test:
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        go: ["1.19"]
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version: ${{ matrix.go }}
+      - name: "test"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PULL_REQUEST_COMMENT_TEST: true
+        run: |
+          go test -cover -coverprofile coverage.txt -race -v ./...
+      - uses: codecov/codecov-action@v3
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
yes! as you mentioned we can remove this workflow from here as we already have to workflows at the root of the repo (this will simplify the example)

As next steps:
- you can remove this workflow
- I'll create another ticket so you can add it as an example on devops-reference where we have some [examples using github actions](https://github.com/nanlabs/devops-reference#github-actions) if you want 😊 
- I'll create another PR later to fix the current CI as soon as we merge this PR. You can ignore the failing checks on the PR for now 👌🏻

Reply:
Sounds good! :ok_hand:

Reply:
✔️ Done! Workflow removed.